k8s/templates/studio-deployment.yaml

```diff
@@ -96,20 +82,13 @@ spec:
         tier: workers
     spec:
       volumes:
-      {{- if .Values.minio.externalGoogleCloudStorage.enabled }}
-      - name: gcs-creds
-        secret:
-          secretName: {{ template "studio.fullname" . }}-gcs
-      {{- end }}
+        {{ include "studio.volume.gcs-creds" . | nindent 8 }}
       containers:
       - name: worker
         image: {{ .Values.studioApp.imageName }}
         command:
         - make
         - prodceleryworkers
         volumeMounts:
-        {{- if .Values.minio.externalGoogleCloudStorage.enabled }}
-        - name: gcs-creds
-          mountPath: /secrets/gcs
-        {{- end }}
+        {{ include "studio.pvc.gcs-creds" . | nindent 10 }}
```

Comment:
does it matter if the nindent number for this specific line is 8 or 10?

Reply:
@lyw07 yeah, it's whatever is needed to make it syntactically valid in the final output YAML. I just kept running `helm template -x <yaml>` until it looked right!

Reply:
cool! thanks! everything looks good. I will spin up a demo server then.